ext/drbbdup/drbbdup.h

```diff
@@ -279,9 +281,9 @@ typedef struct {
      */
     void *user_data;
     /**
-     * The maximum number of cases, excluding the default case, that can be associated
-     * with a basic block. Once the limit is reached and an unhandled case is encountered,
-     * control is directed to the default case.
+     * The maximum number of alternative cases, excluding the default case, that can be
```

Comment:
optional: Rename the field to `non_default_case_limit`?

Reply:
Done.